docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,169 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
+## How to use predefined entering animation?
+
+### 1. Import chosen animation
+```js
+    import { Animation } from 'react-native-reanimated';
```

Comment:
This is not a workable code because there is no such thing as Animation in the exported vars from reanimated lib. If we put a code samples it is better that they can actually work or at least we make it clear that this isn't an existing variable and should be replaced with the name of the animation you want to use. I think in this case it'd be better to just say that "for example if we want to use EaseIn animation then..."